cmd/build_test.go

```diff
@@ -210,6 +212,39 @@ func TestBuildCmd(t *testing.T) {
 		assert.NotNil(t, err)
 	})
 
+	t.Run("Success build cmd with --local", func(t *testing.T) {
+		root := newBuildCmd()
+
+		root.SetArgs([]string{"test", "--local"})
+		buf := bytes.NewBuffer(nil)
+		root.SetOut(buf)
+
+		cwd, err := os.Getwd()
+		if err != nil {
+			t.Fatal(err)
+		}
+
+		cnfDir := filepath.Join(cwd, ".sdlocal")
+		os.MkdirAll(cnfDir, 0777)
+		defer os.RemoveAll(cnfDir)
+
+		cnfPath := filepath.Join(cnfDir, "config")
+		err = os.Link("./testdata/config", cnfPath)
+		if err != nil {
+			t.Fatal(err)
+		}
+
+		configNew = func(path string) (config.Config, error) {
+			assert.Equal(t, cnfPath, path)
+			return config.Config{}, nil
+		}
```

Comment:
I think we should write the following:
```suggestion
		defFunc := configNew
		configNew = func(path string) (config.Config, error) {
			assert.Equal(t, cnfPath, path)
			return config.Config{}, nil
		}
		defer func() {
			configNew = defFunc
		}()
```

Reply:
thx suggestion.:bow: